lib/srv/usermgmt.go

```diff
@@ -134,7 +157,12 @@ type HostUserManagement struct {
 	userGrace time.Duration
 }
 
+type HostSudoersManagement struct {
+	backend HostSudoersBackend
+}
+
 var _ HostUsers = &HostUserManagement{}
+var _ HostSudoers = &HostSudoersManagement{}
```

Comment:
nit and style (gofumpt autoformats adjacent `var`s in a single group):

```suggestion
var (
	_ HostUsers   = (*HostUserManagement)(nil)
	_ HostSudoers = (*HostSudoersManagement)(nil)
)
```